aip/general/0162.md

```diff
@@ -160,26 +144,23 @@ message TagBookRevisionRequest {
   succeed and the tag updated to point to the new requested revision ID. This
   allows users to write code against specific tags (e.g. `published`) and the
   revision can change in the background with no code change.
-
 ### Listing revisions
 
-APIs implementing resource revisions **should** provide a custom method for
-listing the revision history for a resource, with a structure similar to
-standard `List` methods ([AIP-132][]):
+APIs implementing resource revisions `should` provide a standard List method for listing the revision history for a resource
 
 ```proto
```

Comment:
proto is unnescessary, I'd like just link to AIP-132 and state that it has to be compatible with AIP-132 get.

Reply:
removed and updated